src/libraries/Microsoft.Extensions.Caching.Memory/src/CacheEntry.cs

```diff
@@ -220,7 +221,8 @@ public void Dispose()
             }
         }
 
-        internal bool CheckExpired(DateTimeOffset now)
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal bool CheckExpired(in DateTimeOffset now)
```

Comment:
How much does the ‘in’ help here?

Reply:
I just wanted to be 100% sure that this struct is not being copied on Linux.

Reply:
> I just wanted to be 100% sure that this struct is not being copied on Linux.

Did this one change make any measurable impact?  We should not be sprinkling lots of `in`s everywhere unless they're actually meaningful.